src/plugins/viewport.ts

```diff
@@ -1,8 +1,8 @@
 import { addEvent, bind } from "../core";
 
 export default class Viewport implements IPlugin {
-  private eventName : string = "Viewport";
-  private distanceThreshold : number = 20;
+  private eventName = "Viewport";
+  private distanceThreshold = 20;
```

Comment:
I agree that this has to be in the config as well - we've discussed it on Sarvesh's PR too, but we'll make a separate PR for that.

---
In reply to: [119988354](https://github.com/Microsoft/clarity-js/pull/16#discussion_r119988354) [](ancestors = 119988354)